llvm/lib/Target/EraVM/EraVMInstrInfo.cpp

```diff
@@ -871,8 +878,7 @@ EraVMInstrInfo::getOutliningType(MachineBasicBlock::iterator &MBBI,
   }
 
   // Don't outline context.gas_left instruction.
-  if (MI.getOpcode() == EraVM::CTXr_se &&
-      getImmOrCImm(MI.getOperand(1)) == EraVMCTX::GAS_LEFT)
+  if (MI.getOpcode() == EraVM::CTXGasLeft)
```

Comment:
As I mentioned: we created separate instruction instead of "contex operand". So, there is no `CTXr_se` and similar instructions anymore (see .td file).